docs/site/reference.md

```diff
@@ -119,12 +119,12 @@ patient data and used during query retrieval. The following blocking key types a
 These are the functions that can be used to evaluate the matching results as a collection, thus
 determining it the incoming payload is a match or not to an existing Patient record.
 
-`func:recordlinker.linking.matchers.eval_perfect_match`
+`func:recordlinker.linking.matchers.match_rule`
```

Comment:
I don't disagree with any of the names, but do you think we should have some sort of pattern for rules and a different one for evaluators?  For example, I know these don't read as well, but its apparent to the user where in the algorithm they are being used.  I'm trying to balance readability of how it works to readability of where its used.

## rules
- rule_match
- rule_probablistic_match

## evaluators
- eval_match_any
- eval_fuzzy_match
- eval_probablistic_fuzzy_match

Reply:
I could go either way. I think you make a good case for the value of indicating to the user where in the algorithm these funcs are being used. If you feel strongly about it, I can add in prefixes. `rule_` sounds good, how about `compare_` or `feature_` for evaluators? I think rule vs. evaluator are too similar that it may be confusing what the difference between these funcs are.

Reply:
Ultimately the API docs, along with our validation steps, should dictate where these functions can be used.  So really the prefixes or suffixes are just an added bonus.  How the function works is the most important thing.  If you can weave those in and you think the names still make sense, awesome.  If its a stretch, keep it the way you have it.

Reply:
Sounds good. Will weave in today.